### PR TITLE
Feature/template config folder

### PR DESCRIPTION
**Pull request by JohannesFinsveen**
Fixes  statisticssweden/PxWeb#124

### CODE REVIEW

File: GetConfig.bat [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@echo off
+set mypath=%~dp0
+echo Folder of executing script %mypath%
+
+
+echo cheching if %mypath%PXWeb\Web.config exists
```
[the remark this answers is not in the document]
Comment (JohannesFinsveen) [reply GetConfig.bat]: Ok :-)

**Review by likp — APPROVED** [review]
Works fine

**Comment by JohannesFinsveen** [comment]
@likp Who should merge it?

**Comment by likp** [comment]
@JohannesFinsveen you could merge it if you like to otherwise you can assign it to me or @MikaelNordberg.

**Pull request merged**